src/function/table/pragma_detailed_profiling_output.cpp

```diff
@@ -0,0 +1,98 @@
+#include "duckdb/function/table/sqlite_functions.hpp"
+
+#include "duckdb/catalog/catalog_entry/table_catalog_entry.hpp"
+#include "duckdb/catalog/catalog_entry/view_catalog_entry.hpp"
+#include "duckdb/planner/constraints/bound_not_null_constraint.hpp"
+#include "duckdb/main/query_profiler.hpp"
+#include "duckdb/main/client_context.hpp"
+
+#include "duckdb/common/limits.hpp"
+namespace duckdb {
+
+struct PragmaDetailedProfilingOutputData : public FunctionOperatorData {
+	explicit PragmaDetailedProfilingOutputData(idx_t rows) : rows(rows) {
+	}
+	idx_t rows;
+};
+
+static unique_ptr<FunctionData> PragmaDetailedProfilingOutputBind(ClientContext &context, vector<Value> &inputs,
+                                                                  unordered_map<string, Value> &named_parameters,
+                                                                  vector<LogicalType> &return_types,
+                                                                  vector<string> &names) {
+	names.emplace_back("OPERATOR_ID");
+	return_types.push_back(LogicalType::INTEGER);
+
+	names.emplace_back("FUNCTION_ID");
+	return_types.push_back(LogicalType::INTEGER);
+
+	names.emplace_back("NAME");
+	return_types.push_back(LogicalType::VARCHAR);
+
+	names.emplace_back("TIME");
+	return_types.push_back(LogicalType::DOUBLE);
+
+	return make_unique<TableFunctionData>();
+}
+
+unique_ptr<FunctionOperatorData> PragmaDetailedProfilingOutputInit(ClientContext &context,
+                                                                   const FunctionData *bind_data,
+                                                                   vector<column_t> &column_ids,
+                                                                   TableFilterCollection *filters) {
+	return make_unique<PragmaDetailedProfilingOutputData>(1024);
```

Comment:
Why is this hardcoded to 1024? Is this meant to be STANDARD_VECTOR_SIZE?

Reply:
Yes.